cookbooks/aws-parallelcluster-config/templates/default/log_rotation/parallelcluster_log_rotation.erb

```diff
@@ -205,5 +225,15 @@
   endscript
   nodateext
 }
+<%# compute bootstrap error log %>
+/var/log/parallelcluster/bootstrap_error_msg {
+  rotate 1
+  size 10M
+  missingok
+  notifempty
+  copytruncate
+  su pcluster-admin pcluster-admin
+  nodateext
+}
```

Comment:
Out of curiosity, given that this log file will only be created right before the bootstrap fails and compute node gets terminated, instead of continuously growing, why is it necessary to have a log rotation mechanism for it? (although I don't think it hurts)

Reply:
After talking with Paolo, this file could be written to on update, so in theory, on the head node, this file's growth could be unbounded.

Reply:
PR to add the configuration for both headnode and compute node